packages/clay-alert/src/index.tsx

```diff
@@ -138,7 +138,7 @@ const ClayAlert: React.FunctionComponent<IClayAlertProps> & {
 		>
 			<ConditionalContainer>
 				<div className="alert-autofit-row autofit-row">
-					<div className="autofit-col">
+					<div className="autofit-col inline-item-before">
```

Comment:
@dgarciasarai, not sure if I'm remembering correctly, but was this what we discussed in the past?

Reply:
@jbalsas this was the same fix we had in the other ticket. https://github.com/liferay/clay/pull/2762/files#diff-ba7147c142a149f7ae7a4b2c0874be71R108

Reply:
Yeah, I was wondering why we didn't apply it in the first place... 🤔 . I thought we told @dgarciasarai she should add that class in her own code, not sure if it's the same case, though